src/components/omdb/Info.jsx

```diff
@@ -1,46 +1,34 @@
-import React, { useState, useEffect } from "react";
-import Button from "react-bootstrap/Button";
-import Form from "react-bootstrap/Form";
-import Card from "react-bootstrap/Card";
+import React, { useState, useEffect, useMemo } from "react";
+import { Card, Button, Form } from "react-bootstrap";
+import { useParams } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
 import axios from "axios";
-import { useSearchParams, useNavigate } from "react-router-dom";
-import Loader from "../Components/Loader";
 
-export const MovieInfo = () => {
-  const [searchParams] = useSearchParams();
+export default function Info() {
+  let { searchId } = useParams();
   const navigate = useNavigate();
   const [movieInfo, setMovieInfo] = useState();
-  const [isLoading, setIsLoading] = useState(true);
 
-  useEffect(() => {
-    if (searchParams.get("token")) {
-      detailShow(searchParams.get("token"));
-    }
-  }, []);
+  useMemo(() => {
+    axios
+      .get(
+        "https://www.omdbapi.com/?apikey=" +
+          import.meta.env.VITE_API_KEY +
+          "&type=movie&plot=full&t=" +
+          searchId
+      )
+      .then((response) => {
+        setMovieInfo(response.data);
+        const timer = setTimeout(() => {}, 1500);
+        return () => clearTimeout(timer);
+      })
+      .catch(function (error) {
+        console.warn(error);
```

Comment:
Don't just simply console log errors. Maybe you can handle it using something like react-toastify. Search to find out more